families/seth/rpc/src/calls/transaction.rs

```diff
@@ -67,11 +67,12 @@ pub fn send_transaction<T>(params: Params, mut client: ValidatorClient<T>) -> Re
         Error::invalid_params("`data` not set")))?;
 
     // Optional Arguments
+    let txn_count = client.get_account(from.clone(), BlockKey::Latest).map(|a| a.unwrap().nonce)?;
```

Comment:
The internal a.unwrap() should be handled as well. (If I have time during hackfest, I'll try and come up with a more concrete suggestion.)

Reply:
See my _next_ update which uses `match` based on the idioms used by other calls to the client later on in the file...!